roles/nuage_master/tasks/main.yaml

```diff
@@ -41,6 +57,41 @@
   become: yes
   copy: src="{{ vsd_user_key_file }}" dest="{{ cert_output_dir }}/{{ vsd_user_key_file | basename }}"
 
+- name: Copy Nuage Master monitor daemon sets yaml to master node
+  become: yes
+  copy:
+    src: nuage-master-config-daemonset.yaml
+    dest: /etc/nuage-master-config-daemonset.yaml
+    mode: 0755
+  when: openshift.common.is_atomic | bool
+
+- name: Spawn Nuage Master monitor daemon sets pod
+  shell: oc create -f /etc/nuage-master-config-daemonset.yaml
+  ignore_errors: true
+  when: openshift.common.is_atomic | bool
+
+- name: Copy Nuage node daemon sets yaml to master node
+  become: yes
+  copy:
+    src: nuage-node-config-daemonset.yaml
+    dest: /etc/nuage-node-config-daemonset.yaml
+    mode: 0755
+  when: openshift.common.is_atomic | bool
+
+- name: Spawn Nuage CNI daemon sets pod
+  shell: oc create -f /etc/nuage-node-config-daemonset.yaml
+  ignore_errors: true
+  when: openshift.common.is_atomic | bool
+
+- name: Restart daemons
```

Comment:
any reason why we are doing master/api server/controller restart after launching the daemon sets?